DEPENDENCIES.md

```diff
@@ -8,7 +8,7 @@
 | clang-format              | >= 8.0           |    All   |                      Yes (formatting) |
 | clang-tidy                | >= 8.0           |    All   |                         Yes (linting) |
 | cmake                     | >= 3.9           |    All   |                                    No |
-| gcc                       | 8.{2,3}          |    All   | Yes, if clang installed, not for OS X |
+| gcc                       | >= 9.0           |    All   | Yes, if clang installed, not for OS X |
```

Comment:
See comment on CMakeLists.txt.

Reply:
Since you do not touch install.sh as well: We implicitly require gcc-9 to be available, assuming the developer has Ubuntu 19.10 installed. Shall we maybe add gcc-9 to install.sh (for older Ubuntu releases) or alternatively make more explicit here in this file that we assume macOS 10.15 or Ubuntu 19.10?

Reply:
LÜGENTHOMAS!

Reply:
Nevermind.